src/Tickets/Emails/Email/Completed_Order.php

```diff
@@ -0,0 +1,402 @@
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$recipient = apply_filters( 'tec_tickets_emails_recipient', $recipient, self::$id, $this->template );
+
+		return $this->format_string( $recipient );
+	}
+
+	/**
+	 * Get default email recipient.
+	 *
+	 * @since TBD
+	 *
+	 * @return string $recipient The default email recipient.
+	 */
+	public function get_default_recipient(): string {
+		return get_option( 'admin_email' );
+	}
+
+	/**
+	 * Get email heading.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email heading.
+	 */
+	public function get_heading(): string {
+		$heading = tribe_get_option( static::$option_heading, $this->get_default_heading() );
+
+		/**
+		 * Allow filtering the email heading for Completed Order.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email heading.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$heading = apply_filters( 'tec_tickets_emails_completed_order_heading', $heading, self::$id, $this->template );
+
+		/**
+		 * Allow filtering the email heading globally.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email heading.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$heading = apply_filters( 'tec_tickets_emails_heading', $heading, self::$id, $this->template );
+
+		return $this->format_string( $heading );
+	}
+
+	/**
+	 * Get default email heading.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_default_heading() {
+		return esc_html__( 'Completed order: #{order_number}', 'event-tickets' );
+	}
+
+	/**
+	 * Get email subject.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email subject.
+	 */
+	public function get_subject(): string {
+		$subject = tribe_get_option( static::$option_subject, $this->get_default_subject() );
+
+		$subject = $this->format_string( $subject );
+
+		// @todo: Probably we want more data parsed, or maybe move the filters somewhere else as we're always gonna
+		// apply filters on the subject maybe move the filter to the parent::get_subject() ?
+
+		/**
+		 * Allow filtering the email subject for Completed Order.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $subject  The email subject.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$subject = apply_filters( 'tec_tickets_emails_completed_order_subject', $subject, self::$id, $this->template );
+
+		/**
+		 * Allow filtering the email subject globally.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $subject  The email subject.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$subject = apply_filters( 'tec_tickets_emails_subject', $subject, self::$id, $this->template );
```

Comment:
The same comment about the filter naming and parameter as above. We're repeating these all over, and we'll be doing it in every email that we'll register. If we're doing a generic filter we should centralize it.